deploy/olm-catalog/bundle/manifests/migration.openshift.io_migclusters.yaml

```diff
@@ -47,19 +48,26 @@ spec:
           description: MigClusterSpec defines the desired state of MigCluster
           properties:
             azureResourceGroup:
+              description: AzureResourceGroup is for azure clusters -- it's the resource group that in-cluster volumes use.
               type: string
             caBundle:
+              description: If the migcluster needs SSL verification for connections a user can supply a custom CA bundle.
               format: byte
               type: string
             exposedRegistryPath:
+              description: Stores the path of registry route when using direct migration.
               type: string
             insecure:
+              description: Insecure stores the status of the connection with source cluster. True signifies the connection is insecure.
               type: boolean
             isHostCluster:
+              description: IsHostCluster specifies if the cluster is host (where the controller is installed) or not. This is a required field.
               type: boolean
             refresh:
+              description: Refresh True forces the controller to run a connection test on migcluster.
               type: boolean
             restartRestic:
+              description: RestartRestic is an override setting to tell the controller that the source cluster restic needs to be restarted after stage pod creation.
```

Comment:
I'm not sure we need to mention the var name given that each of these descriptions is nested right under the var.

E.g. I'm thinking it would make sense to change:
```
RestartRestic is an override setting  [...]
```
-->
```
An override setting [...]
```

Reply:
Yip! Makes sense to remove the variable name from the description.